taskDetails.php

```diff
@@ -1,31 +1,91 @@
+                <?php
+                    require("conn.php"); 
+                    
+                    $query = "SELECT listID,listTitle FROM taskList";
+                    $taskLists = mysqli_query($conn, $query);
+                    while ($row = mysqli_fetch_array($taskLists)) {
+                        echo "<option value=" . $row['listID'] . ">" . $row['listTitle'] . "</option>"; 
+                    }
+                ?> 
+            </select>
+            </div>
+        <div class="col">
+            <label for="taskEffort"> Effort</label>
+           <input style="height:50px;width:80px;font-size:14pt;"type="number" class="form-control" name="taskEffort">
+           <small id="" class="form-text text-muted">Amount of work required</small>
+        </div>
+
+        <div class="col">
+            <label for="taskTypeWork"> Type of Work</label>
+           <input style="height:50px;width:160px;font-size:14pt;"type="number" class="form-control" name="taskTypeWork">
+           <small id="" class="form-text text-muted">Detail of work type</small>
+        </div>
+        <div class="col">
+            <label for="taskDateDue"> Due Date</label>
+            <input style="height:50px;width:150px;font-size:14pt;"type="number" class="form-control" name="taskDateDue">
+            <small id="" class="form-text text-muted">Due date of task</small>
+        </div>
+    </div>
+    </div>
+        <div class="form-group form-check">
+
+        </div>
+        <button type="submit" class="btn btn-primary" name="submit">Submit</button>
+    </form>
+
+
+
+</head>
 <?php
 //This file contains the php code for inserting tast detail data from the html form
 
 //declaring local php variables 
-$submitted_tastName = $_POST['taskTitle'];
+$submitted_taskName = $_POST['taskTitle'];
 $submitted_taskDetail = $_POST['tastDecrip'];
 $submitted_taskEffort = $_POST['taskEffort'];
 $submitted_taskDateDue= $_POST['taskDateDue'];
 $submitted_taskTypeWork= $_POST['taskTypeWork'];
-
-
-//binding php var to sql object
-echo $submitted_tastName;
-echo $submitted_taskDetail;
-echo $submitted_taskEffort;
-echo $submitted_taskDateDue;
-echo $submitted_taskTypeWork;
+$submitted_taskListAssigned = $_POST['listID'];
 
 //imports the objects and valiues from conn
 require("conn.php"); 
-$sql = "INSERT INTO `task` (`taskID`, `taskTitle`, `description`, `dueDate`, `taskDateCreated`, `importance`, `typeOfWork`, `c.courseID`, `tl.listID`, `archived`) VALUES (null, '$submitted_tastName', '$submitted_taskDetail', $submitted_taskDateDue , '10', $submitted_taskEffort, '$submitted_taskTypeWork', '111', '1', '1');";
+$sql = "INSERT INTO `task` (`taskID`, `taskTitle`, `description`, `dueDate`, `taskDateCreated`, `importance`, `typeOfWork`, `c.courseID`, `tl.listID`, `archived`) 
+        VALUES (null, '$submitted_tastName', '$submitted_taskDetail', $submitted_taskDateDue , '10', $submitted_taskEffort, '$submitted_taskTypeWork', '111', $submitted_taskListAssigned , '1');";
```

Comment:
$submitted_tastName should be changed to $submitted_taskName

Reply:
Thanks for the catch! will update asap.